pkg/column-mapping/README.md

```diff
@@ -29,11 +29,19 @@ type Rule struct {
 now we support following expressions
 
 ``` go
-addPrefix
-addSuffix
-clone
+add prefix, with arguments[prefix]
+
+add suffix, with arguments[suffix]
+
+partition id, with arguments [instance_id, prefix of schema, prefix of table]
+[1:1 bit][2:9 bits][3:10 bits][4:44 bits] int64  (using default bits length)
+- 1: useless, no reason
+- 2: schema ID (schema suffix)
+- 3: table ID (table suffix)
+- 4: origin ID (>= 0, <= 17592186044415)
+And schema = arguments[1] + schema suffix, table = arguments[2] + table suffix
```

Comment:
I didn't understand this until I read `computeID` :joy:
maybe you can give an example